src/CustomItem.cs

```diff
@@ -51,31 +49,44 @@ public CustomItem(
 		// C# pls
 		if (soldOnlyAt == default(List<Shop>)) { soldOnlyAt = new List<Shop>(); }
 		Main.Log($"Instantiating {itemInfo.Name}");
+
+		// save item info for later use
+		//this.itemInfo = itemInfo;
 		
 		Name = itemInfo.Name;
 		Description = itemInfo.Description;
-		ItemPrefab = CreateItemPrefab(unityPrefab);
-		ProvidedPrefab = unityPrefab;
+		ItemPrefab = CreateItemPrefab(providedItemPrefab);
+		//this.providedItemPrefab = providedItemPrefab;
 
 		Main.Log($"Loaded prefabs for {itemInfo.Name}");
 
-		var previewBounds = itemInfo.PreviewBounds;
-		Main.Log($"Preview bounds: {previewBounds}");
-		if (previewBounds == default)
-		{
-			previewBounds = new Vector3(0.35f, 0.3f, 0.2f);
-		}
+		var previewBounds = new Vector3(0.35f, 0.3f, 0.2f);
 		if (previewRotation == default) { previewRotation = Vector3.zero; }
```

Comment:
Something's gone wrong here, this does not compile. 

```
0>CustomItem.cs(64,7): Error CS0103 : The name 'previewRotation' does not exist in the current context
0>CustomItem.cs(64,37): Error CS0103 : The name 'previewRotation' does not exist in the current context
```

Reply:
This was a merge artifact - it's been cleaned up, that line shouldn't even still exist.